lighthouse-core/audits/accessibility/aria-valid-attr.js

```diff
@@ -33,6 +33,8 @@ class ARIAValidAttr extends AxeAudit {
       category: 'Accessibility',
       name: 'aria-valid-attr',
       description: 'Element aria-* attributes are valid and not misspelled or non-existent.',
+      helpText: 'Screen readers may not be able to read elements with invalid ARIA ' +
```

Comment:
Suggest rewording to something like "Assistive technology, such as a screen reader, will not be able to interpret ARIA attributes with invalid attribute names"